docs/data_warehouse.json

```diff
@@ -1,403 +1,448 @@
ates a single report. To use the example, replace 'request.rsid' with an rsid to which you have Data Warehouse access (shown as 'sistr2' in the example), and then add a valid 'exportLocationUUID' under delivery.\n\n```json\n{\r\n  \"schedule\": {\r\n    \"periodSettings\": {\r\n      \"frequency\": \"runOnceSetPeriod\"\r\n    }\r\n  },\r\n  \"request\": {\r\n    \"name\": \"Run-Once Example\",\r\n      \"reportParameters\": {\r\n          \"reportRange\": {\r\n              \"startDateTime\": \"2023-01-01T00:00:00Z\",\r\n              \"endDateTime\": \"2023-01-01T23:59:59Z\"\r\n          },\r\n          \"dimensionList\": [\r\n          ],\r\n          \"metricList\": [\r\n              {\r\n                  \"id\": \"metrics/visitors\"\r\n              }\r\n          ]\r\n      },\r\n    \"rsid\": \"sistr2\"\r\n  },\r\n  \"delivery\": {\r\n      \"exportLocationUUID\": \"00000000-0000-0000-0000-000000000000\",\r\n      \"email\": {\r\n          \"notificationEmailTo\": \"donotuse@adobe.com\"\r\n      }\r\n  }\r\n}\r\n```\n\n Below is an example of a request that generates a daily report (for yesterday's data) for 10 days.\n\n```json\n{\r\n  \"schedule\": {\r\n    \"periodSettings\": {\r\n      \"frequency\": \"daily\"\r\n    }, \"cancelSettings\": {\r\n      \"cancelMethod\": \"afterOccurrences\",\r\n      \"endAfterNumOccurrences\": 10\r\n    }\r\n  },\r\n  \"request\": {\r\n    \"name\": \"Run Daily Example\",\r\n      \"reportParameters\": {\r\n          \"reportRange\": {\r\n              \"preset\": \"Yesterday\"\r\n          },\r\n          \"dimensionList\": [\r\n          ],\r\n          \"metricList\": [\r\n              {\r\n                  \"id\": \"metrics/visitors\"\r\n              }\r\n          ]\r\n      },\r\n    \"rsid\": \"sistr2\"\r\n  },\r\n  \"delivery\": {\r\n      \"exportLocationUUID\": \"00000000-0000-0000-0000-000000000000\",\r\n      \"email\": {\r\n          \"notificationEmailTo\": \"donotuse@adobe.com\"\r\n      }\r\n  }\r\n}\r\n```\n\nThe example below shows how to specify participation as a metric in the 'metricList', instead of 'visitors'. as shown in the previoius example. The 'id' of items in the 'metricList' are specified as identity service names.\n\n```json\n{\r\n  \"id\": \"metrics/campaigninstances\",\r\n  \"properties\": {\r\n     \"isParticipation\": true\r\n   }\r\n}\r\n\r\n```\n\nFor 'dimensionList', the entries can reference an identity service or a segment service. The 'id' in the following 'dimensionList' example is specified as the path of an identity service.\n\n```json\n{\r\n  \"id\": \"variables/geocity\",\r\n}\r\n\r\n```\n\nThe example below shows a 'dimensionList' entry that references a segment service, instead of the identity service shown in the previous example. Note that when a 'namespace' is not specified with a service, it is assumed to be an identity service.\n\n```json\n{\r\n  \"namespace\": \"Segment Service\",\r\n  \"id\": \"s0_000000000000000000000000\",\r\n}\r\n\r\n```\n\n",
```

Comment:
@johnwight One of the lines I have a question about is the change of `The 'id' of items in the metricList are identity service names. Below is an example of what an entry in metricsList would look like to use the participation metric instead.` to `The example below shows how to specify participation as a metric in the 'metricList', instead of 'visitors'. as shown in the previoius example. The 'id' of items in the 'metricList' are specified as identity service names.`. The new language doesn't match my understanding of a "participation metric" (https://experienceleague.adobe.com/docs/analytics-platform/using/cja-components/cja-calcmetrics/cm-workflow/participation-metric.html?lang=en). The issue is that participation, itself, isn't the metric, but it is a type of metric. However, I do recognize my own short-comings in being able to accurately describe some of the technical features we expose to customers here at Adobe. What are your thoughts? Thanks.